var/spack/repos/builtin/packages/bacio/package.py

```diff
@@ -36,7 +39,9 @@ class Bacio(CMakePackage):
     variant("pic", default=True, description="Build with position-independent-code")
 
     def cmake_args(self):
+        variant("shared", default=False, description="Build shared library", when="@2.6.0:")
```

Comment:
The `variant` call should go right after the existing one (`variant("pic"...`), in the main body of the Bacio class definition.

Reply:
OK, thanks!